pkg/status/management_conflict_error.go

```diff
@@ -53,20 +55,22 @@ func ManagementConflictErrorWrap(resource client.Object, newManager string) Mana
 }
 
 type managementConflictErrorImpl struct {
-	underlying Error
-	resource   client.Object
-	// newManager refers to the manager annotation for the current remediator/reconciler.
```

Comment:
Maybe keep the comments or update them for change of definition.

Reply:
The methods have descriptions now. The definition didn't really change, just the names. 

The comments here are actually specific to the -Wrap constructor inputs, not required or used by the impl itself. The impl shouldn't really care about the annotation or where the inputs came from.

That said, I see that technically speaking, the Invert() method does pull the current manager from the object, which it probably shouldn't. it should use the field instead. And we could probably make this slightly more efficient by not storing the whole Object and only storing the ID instead. THat would completely remove the annotation bit from the interface.

Reply:
Ah, nevermind. We can't replace the object with just a ID, because the Body() formatter calls PrintResource, which reads the source annotation. I guess we could make an object-like thing that includes annotations, but that'd be a larger refactor.

I can fix Invert() though

Reply:
Done.